ansible/roles/dashd/tasks/main.yml

```diff
@@ -96,5 +96,9 @@
   retries: 5
   delay: 10
 
+- name: import faucet privkey
+  when: inventory_hostname in groups['faucet_wallet']
+  shell: dash-cli importprivkey {{ faucet_privkey }}
```

Comment:
Are you sure this works before the wallet is created?

Reply:
Oh that's a good catch, I've moved it below the "create_wallet" section

Reply:
So it should work on both <18.2 and >18.2 now